data/frame_json_test.go

```diff
@@ -406,153 +417,3 @@ func TestFrame_UnmarshallUint64(t *testing.T) {
 	}
 	require.EqualValues(t, []uint64{math.MaxUint64, math.MaxUint64, 0, 1, 2, 3, 4, 5}, values)
 }
-
-// This function will write code to the console that should be copy/pasted into frame_json.gen.go
-// when changes are required. Typically this function will always be skipped.
-func TestGenerateGenericArrowCode(t *testing.T) {
```

Comment:
🎉

Reply:
@asimpson a feature toggle was used to flip between an unmodified copy of the prometheus datasource framing code and a replacement. once the new code was determined to be ok, the old framing code was deleted